js/app.js

```diff
@@ -206,29 +207,26 @@ var TwineApp = Backbone.Marionette.Application.extend(
       }, { wait: true });
 
       // and child passages
-      
-      $story.find(selectors.passageData).each(function()
-      {
+
+      var createPassagesFromData = function() {
         var $passage = $(this);
         var id = $passage.attr('pid');
         var pos = $passage.attr('position');
         var posBits = pos.split(',');
-        var tags = $passage.attr('tags').trim();
+        var tags = $passage.attr('tags') ? $passage.attr('tags').trim() : "";
         tags = tags === "" ? [] : tags.split(/\s+/);
 
-        var passage = allPassages.create(
-        {
-          name: $passage.attr('name'),
-          tags: tags,
-          text: $passage.text(),
-          story: story.id,
-          left: parseInt(posBits[0]),
-          top: parseInt(posBits[1])
-        }, { wait: true }); 
-
-        if (id == startPassageId)
-          story.save({ startPassage: passage.id });
-      });
+        var passageObj = createPassage($passage, story.id);
+
+        var passageModel = allPassages.create(passageObj, { wait: true });

```

Comment:
1. Yea, this was me not trying to rock the boat too much. Previously we saw in developing importsms.js that we could do something like...
```
var passageCollection = new PassageCollection();
passageCollection.create(passageObj, {wait: true}); 
```
... and that would work. So we could technically do that here too. I just kept it the way it was to not change more than I needed to. But I'm sort of ambivalent now to changing/fixing how that's done.

2. If you look in `PassageCollection.model` you'll see it creating the different kinds of models based on the type. The [docs](http://backbonejs.org/#Collection-model) tell us that this is used in `Collection.add`, `.create`, and `.reset`.